gnmi_server/server.go

```diff
@@ -376,6 +382,18 @@ func (s *Server) Get(ctx context.Context, req *gnmipb.GetRequest) (*gnmipb.GetRe
 	return &gnmipb.GetResponse{Notification: notifications}, nil
 }
 
+// SaveOnSetEnabled saves configuration to a file
+func SaveOnSetEnabled() {
+	if err := transformer.SaveStartupConfig(); err != nil {
```

Comment:
Should not invoke transformer functions directly from here. Always go through transl and translib.

Reply:
Done